### ISSUE TITLE
Cleaner class declaration

### ISSUE THREAD

**Issue by benbieler** [Sententiaregum/Sententiaregum#244]
### Description

A component like the following: 

``` js

export default class HelloWorld extends React.Component {
   ...
}
```

could also be declared like this:

``` js
export default class HelloWorld extends Component {
   ...
}
```

But prior to that, the import statement has to be adjusted:

``` js
import React, {Component} from 'react';
```
### Further fixes

``` javascript
import { render } from 'react-dom';
render((<h1></h1>));  // instead of ReactDOM.render()
```

``` javascript
import { expect } from 'chai';
expect(true).to.equal(true);
```

``` javascript
import { spy, stub } from 'sinon';
// ...
```

**Comment by Ma27** [comment]
:+1: 

btw instead of `ReactDOM.render` we could do this:

``` javascript

import { render } from 'react-dom';
render((
  <h1>Hello World!</h1>
));
```

and `React.PropTypes` should be fixed the same way :)

**Comment by Ma27** [comment]
assigned to you and added a milestone

**Comment by Ma27** [comment]
IMHO we should wait until #35 is fixed

**Comment by Ma27** [comment]
the `chai.expect` might also be fixed.

**Comment by Ma27** [comment]
I've edited the comment and added more places where this issue exists. I think that before fixing it you should gather a detailed list of places where we use the `import` operator in a wrong way before fixing it. Furthermore the issue with `React.Component` will be hidden for now since the `flux-container` contains its own component base class and this will be included in #35 , so the amount of issues will decrease. /cc @benbieler

**Comment by benbieler** [comment]
@Ma27  And https://github.com/Sententiaregum/Sententiaregum/blob/master/src/Frontend/store/ListenableStore.js#L13 cound be declared like this: {EventEmitter} the you can simply extend EventEmitter

**Comment by Ma27** [comment]
@benbieler as the JS code is quite stable now, we could talk about it again (what needs fix, etc.) as the actual issue you reported months ago is still valid, relevant and definetly correct.

**Comment by benbieler** [comment]
Ok, great!

**Comment by Ma27** [comment]
OK, what I've found so far:
- some components (e.g. the `Menu` component) still use `React.Component`
- when including the `Navbar` in `MenuWrapper`, something like `Navbar.Header` or `Navbar.Brand` will be used. It might be cleaner to include `Navbar` itself and all the subcomponents like `Header` or `Brand`
- instead of `props.n` we could create variables like `const { n } = props;` (e.g. `DropDownItem`)
- instead of using `counterpart` directly (`import counterpart from 'counterpart'`) we could do sth. like `import { registerTranslations} from 'counterpart';`, every JS file using counterpart contains this issue
- instead of `axios.get` we could do `get` and that's it
- instead of `Cookies.get` we could do `get` (`LocaleService`)
- some old tests still use `chai.expect`. `expect` is cleaner and better
- some old tests still use `sinon.*`. using the functions directly is cleaner

Furthermore it might be worth investigating whether or not the usage of `ApiKey` and `Locale` can be simplified to including their functions:

`import { isLoggedIn } from './util/http/ApiKeyService';` rather than `import ApiKey from './util/http/ApiKeyService'`

**Comment by Ma27** [comment]
closed in #348
